moped-database/migrations/1637823185562_alter_project_list_view/up.sql

```diff
@@ -1,3 +1,5 @@
+DROP VIEW "public"."project_list_view";
```

Comment:
I know this is somewhat harmless, but it feels wrong. production and main should be identical. Not to mention migrations do not get re-run if they have been already applied.

Reply:
yeah it feels weird to me too. Apparently production has had a migration since June that main hasn't had. What is another way to fix it? We need to drop a column from the view.

Reply:
Could we apply this line just to the new migration that hasn't been run in prod yet? `1638905763818_run_sql_migration/up.sql`

Reply:
We had a couple good ways to fix it, one would have been ideally a migration that is friendly to both main and production and does the exact same thing. Also, we always have the option of using hasura's cli.

Again, my only problem is that we should make production like a mirror of main, and this does the opposite. If we continue, we are gonna find ourselves troubleshooting conflicts in code and in the database.